app/views/checkouts/_checkout.html.erb

```diff
@@ -36,6 +36,10 @@
         <dt class="col-5">Recalled on:</dt>
         <dd class="col-5"><%= l(checkout.recalled_date, format: :short) %></dd>
       <% end %>
+      <% if !checkout.overdue? %>
+        <dt class="col-5">Remaining:</dt>
+        <dd class="col-5"><%= checkout.days_remaining %> days</dd>
```

Comment:
We should use either:
- i18n's one/other support, or
- [`pluralize`](https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize)

to make sure this says `1 day` when appropriate

Reply:
Used the `pluralize` approach for now. I'd rather migrate things i18n keys in one PR (soon!) and be thoughtful about how we're organizing them.